patches/keboola-csv-src-keboola-csv-csvfile-php.patch

```diff
@@ -0,0 +1,50 @@
+--- /dev/null
```

Comment:
nejde použít novější verze, protože jí nedovolí sapi client, který tu je kvůli migracím configu, takže je potřeba patchnout aby to bylo PHP8 compatible